AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -306,6 +306,13 @@ extern NSInteger const ASDefaultDrawingPriority;
  */
 @property (nonatomic, readonly, assign) ASSizeRange constrainedSizeForCalculatedLayout;
 
+/**
+ * @abstract Sublcass hook for nodes that are acting as root nodes. This method get's called if one of the subnodes
```

Comment:
Minor - "gets" instead of "get's".  Might be better as "This method is called if..."